dash/dash.py

```diff
@@ -703,13 +707,10 @@ def serve_component_suites(self, package_name, path_in_package_dist):
                 )
             )
 
-        mimetype = (
-            {
-                "js": "application/javascript",
-                "css": "text/css",
-                "map": "application/json",
-            }
-        )[path_in_package_dist.split(".")[-1]]
+        resource_path = os.path.dirname(sys.modules[package_name].__file__)
+        full_path = os.path.join(resource_path, path_in_package_dist)
```

Comment:
Will update this implementation to get the mapped mimetype from the `mimetypes` mapping instead of requiring the fullpath.